tendrl/ceph_integration/sds_sync/__init__.py

```diff
@@ -137,7 +137,7 @@ def _sync_rbds(self):
                     size=v['size'],
                     pool_id=pool_id,
                     flags=v['flags'],
-                    provisioned=self._to_bytes(v['provisioned']),
+                    provisioned=self._to_bytes(v.get("provisioned", 0)),
```

Comment:
@anmolsachan I feel just `v.get("provisioned")` you should try and it should work. The only thing in sds_sync take care properly for `None` case.